official/transformer/compute_bleu.py

```diff
@@ -98,32 +100,42 @@ def bleu_wrapper(ref_filename, hyp_filename, case_sensitive=False):
   return metrics.compute_bleu(ref_tokens, hyp_tokens) * 100
 
 
-def main(unused_argv):
-  if FLAGS.bleu_variant is None or "uncased" in FLAGS.bleu_variant:
-    score = bleu_wrapper(FLAGS.reference, FLAGS.translation, False)
+def compute_bleu(flags_obj):
+  """Print out the BLEU scores calculated from the files defined in flags."""
+  if flags_obj.bleu_variant in ("both", "uncased"):
+    score = bleu_wrapper(flags_obj.reference, flags_obj.translation, False)
     print("Case-insensitive results:", score)
```

Comment:
I think a following change will probably needed to change all the print to tf.logging for consistency.